Library/Formula/rtf2html.rb

```diff
@@ -0,0 +1,17 @@
+class Rtf2html < Formula
+  homepage "http://sourceforge.net/projects/rtf2html-lite/"
+  url "https://downloads.sourceforge.net/project/rtf2html-lite/RTF%202%20HTML%20Lite%20-%20Sources/2008-10-03%2C%20first%20LGPL%20release/rtf2html-lite-20081003.tar.bz2"
+  sha1 "eec3b29633f44290fcab3acd5336462cfc737140"
+  version "20081003"
```

Comment:
That's not the case: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/19909/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_rtf2html___strict/